### PR TITLE
Fix validation errors 1

### PR DESCRIPTION
**Pull request by anara123**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
related: https://github.com/demisto/etc/issues/23712

## Description
Fix some of the validation errors in the integrations and scripts

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [ ] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

**Comment by anara123** [comment]
I fixed all the possible quick fixes in the following files.
There are more stuff to fix, that require code changes or validation ignore, I mentioned that in the original issue.
Some of the errors happens because of the demisto-sdk validation problems, also mentioned in the original issue.
Secrets are failing because of a known bug in demisto-sdk, there is already a PR that handles that - not related to this PR, because I didn't touched any secrets.

File: Packs/Legacy/Integrations/ANYRUN/ANYRUN.yml [hunk 1]
```diff
@@ -12,12 +12,12 @@ configuration:
   name: credentials
   required: true
   type: 9
-- defaultvalue: 'true'
+- defaultvalue: ''
```
Comment (Itay4) [inline Packs/Legacy/Integrations/ANYRUN/ANYRUN.yml:15]:
```suggestion
```

File: Packs/Legacy/Integrations/EclecticIQ/EclecticIQ.yml [hunk 2]
```diff
@@ -17,12 +17,12 @@ configuration:
   defaultvalue: ""
   type: 9
   required: true
-- display: Trust any certificate (unsecure)
+- display: Trust any certificate (not secure)
   name: insecure
-  defaultvalue: "true"
+  defaultvalue: ''
```
Comment (Itay4) [inline Packs/Legacy/Integrations/EclecticIQ/EclecticIQ.yml:22]:
```suggestion
```

File: Packs/Legacy/Integrations/Jask/Jask.yml [hunk 3]
```diff
@@ -27,7 +27,7 @@ configuration:
   name: insecure
   required: false
   type: 8
-- defaultvalue: 'true'
+- defaultvalue: ''
```
Comment (Itay4) [inline Packs/Legacy/Integrations/Jask/Jask.yml:30]:
```suggestion
```

File: Packs/Legacy/Integrations/SNDBOX/SNDBOX.yml [hunk 4]
```diff
@@ -20,10 +20,10 @@ configuration:
     required: true
   - display: Use system proxy settings
     name: proxy
-    defaultvalue: "False"
+    defaultvalue: ''
```
Comment (Itay4) [inline Packs/Legacy/Integrations/SNDBOX/SNDBOX.yml:23]:
```suggestion
```

File: Packs/Legacy/Integrations/SNDBOX/SNDBOX.yml [hunk 5]
```diff
@@ -20,10 +20,10 @@ configuration:
...
     type: 8
     required: false
-  - display: Trust any certificate (unsecure)
+  - display: Trust any certificate (not secure)
     name: insecure
     defaultvalue: ''
```
Comment (Itay4) [inline Packs/Legacy/Integrations/SNDBOX/SNDBOX.yml:28]:
```suggestion
```

File: Packs/Legacy/Integrations/integration-Office_EWS.yml [hunk 6]
```diff
@@ -51,7 +51,7 @@ configuration:
   defaultvalue: ""
   type: 8
   required: false
-- display: Trust any certificate (unsecure)
+- display: Trust any certificate (not secure)
   name: insecure
   defaultvalue: ""
```
Comment (Itay4) [inline Packs/Legacy/Integrations/integration-Office_EWS.yml:56]:
```suggestion
```

File: Packs/Legacy/Integrations/integration-Office_EWS.yml [hunk 7]
```diff
@@ -51,7 +51,7 @@ configuration:
...
```
Comment (Itay4) [inline Packs/Legacy/Integrations/integration-Office_EWS.yml:51]:
```suggestion
```

File: Packs/Legacy/Integrations/integration-PhishTank.yml [hunk 8]
```diff
@@ -14,7 +14,7 @@ configuration:
   defaultvalue: "true"
   type: 8
   required: false
-- display: Trust any certificate (unsecure)
+- display: Trust any certificate (not secure)
   name: insecure
   defaultvalue: ""
```
Comment (Itay4) [inline Packs/Legacy/Integrations/integration-PhishTank.yml:19]:
```suggestion
```

File: Packs/Legacy/Integrations/integration-Pipl.yml [hunk 9]
```diff
@@ -18,7 +18,7 @@ configuration:
   defaultvalue: ""
```
Comment (Itay4) [inline Packs/Legacy/Integrations/integration-Pipl.yml:18]:
```suggestion
```

File: Packs/Legacy/Integrations/integration-ProtectWise.yml [hunk 10]
```diff
@@ -30,7 +30,7 @@ configuration:
   name: token
   type: 4
   required: false
-- display: Trust any certificate (unsecure)
+- display: Trust any certificate (not secure)
   name: insecure
   defaultvalue: ""
```
Comment (Itay4) [inline Packs/Legacy/Integrations/integration-ProtectWise.yml:35]:
```suggestion
```

File: Packs/Legacy/Integrations/integration-ProtectWise_3_1_0.yml [hunk 11]
```diff
@@ -22,7 +22,7 @@ configuration:
   defaultvalue: ""
   type: 4
   required: false
-- display: Trust any certificate (unsecure)
+- display: Trust any certificate (not secure)
   name: insecure
   defaultvalue: ""
```
Comment (Itay4) [inline Packs/Legacy/Integrations/integration-ProtectWise_3_1_0.yml:27]:
```suggestion
```

File: Packs/Legacy/Integrations/integration-Remedy-On-Demand.yml [hunk 12]
```diff
@@ -32,7 +32,7 @@ configuration:
   defaultvalue: ""
   type: 8
   required: false
-- display: Trust any certificate (unsecure)
+- display: Trust any certificate (not secure)
   name: insecure
   defaultvalue: ""
```
Comment (Itay4) [inline Packs/Legacy/Integrations/integration-Remedy-On-Demand.yml:37]:
```suggestion
```

File: Packs/Legacy/Integrations/integration-Remedy-On-Demand.yml [hunk 13]
```diff
@@ -32,7 +32,7 @@ configuration:
...
```
Comment (Itay4) [inline Packs/Legacy/Integrations/integration-Remedy-On-Demand.yml:32]:
```suggestion
```

File: Packs/Legacy/Integrations/integration-Sandblast.yml [hunk 14]
```diff
@@ -27,7 +27,7 @@ configuration:
   defaultvalue: ""
   type: 4
   required: false
-- display: Trust any certificate (unsecure)
+- display: Trust any certificate (not secure)
   name: insecure
   defaultvalue: ""
```
Comment (Itay4) [inline Packs/Legacy/Integrations/integration-Sandblast.yml:32]:
```suggestion
```

File: Packs/Legacy/Integrations/integration-Sandblast.yml [hunk 15]
```diff
@@ -27,7 +27,7 @@ configuration:
...
```
Comment (Itay4) [inline Packs/Legacy/Integrations/integration-Sandblast.yml:27]:
```suggestion
```

File: Packs/Legacy/Integrations/integration-Sandblast_3_1_0.yml [hunk 16]
```diff
@@ -27,7 +27,7 @@ configuration:
   defaultvalue: ""
   type: 4
   required: false
-- display: Trust any certificate (unsecure)
+- display: Trust any certificate (not secure)
   name: insecure
   defaultvalue: ""
```
Comment (Itay4) [inline Packs/Legacy/Integrations/integration-Sandblast_3_1_0.yml:32]:
```suggestion
```

File: Packs/Legacy/Integrations/integration-Sandblast_3_1_0.yml [hunk 17]
```diff
@@ -27,7 +27,7 @@ configuration:
...
```
Comment (Itay4) [inline Packs/Legacy/Integrations/integration-Sandblast_3_1_0.yml:27]:
```suggestion
```

File: Packs/Legacy/Integrations/integration-SecureWorks.yml [hunk 18]
```diff
@@ -20,7 +20,7 @@ configuration:
   defaultvalue: ""
```
Comment (Itay4) [inline Packs/Legacy/Integrations/integration-SecureWorks.yml:20]:
```suggestion
```

File: Packs/Legacy/Integrations/integration-SentinelOne.yml [hunk 19]
```diff
@@ -23,7 +23,7 @@ configuration:
   defaultvalue: "2.0"
   type: 0
   required: false
-- display: Trust any certificate (unsecure)
+- display: Trust any certificate (not secure)
   name: insecure
   defaultvalue: "true"
```
Comment (Itay4) [inline Packs/Legacy/Integrations/integration-SentinelOne.yml:28]:
```suggestion
```

File: Packs/Legacy/Integrations/integration-Skyformation.yml [hunk 20]
```diff
@@ -17,7 +17,7 @@ configuration:
   defaultvalue: ""
   type: 9
   required: true
-- display: Trust any certificate (unsecure)
+- display: Trust any certificate (not secure)
   name: insecure
   defaultvalue: ""
```
Comment (Itay4) [inline Packs/Legacy/Integrations/integration-Skyformation.yml:22]:
```suggestion
```

File: Packs/Legacy/Integrations/integration-Skyformation.yml [hunk 21]
```diff
@@ -17,7 +17,7 @@ configuration:
...
```
Comment (Itay4) [inline Packs/Legacy/Integrations/integration-Skyformation.yml:17]:
```suggestion
```

File: Packs/Legacy/Integrations/rasterize_old/rasterize.yml [hunk 22]
```diff
@@ -5,10 +5,10 @@ commonfields:
 configuration:
 - display: 'Return Errors '
   name: with_error
-  defaultvalue: "false"
+  defaultvalue: ''
```
Comment (Itay4) [inline Packs/Legacy/Integrations/rasterize_old/rasterize.yml:8]:
```suggestion
```

File: Packs/Legacy/Integrations/rasterize_old/rasterize.yml [hunk 23]
```diff
@@ -5,10 +5,10 @@ commonfields:
...
   type: 8
   required: false
-- defaultvalue: 'true'
+- defaultvalue: ''
```
Comment (Itay4) [inline Packs/Legacy/Integrations/rasterize_old/rasterize.yml:11]:
```suggestion
```

**Pull request merged**